### ISSUE TITLE
"Vector too long"

### ISSUE THREAD

**Issue by MythicalChu** [Fannovel16/ComfyUI-MotionDiff#11]
It errors during the optimizing step 1/2: camera translation and body orientation
I'm on windows, amd, rx 580 (8Gb Vram), 16Gb ram. No install issues. I'm on the full_motiondiff_example.json, unchanged. Changing parameters don't seem to make a difference. Immediate crash on the start of that step

vector too long

File "C:\AI\ComfyUI_windows_portable\ComfyUI\execution.py", line 153, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "C:\AI\ComfyUI_windows_portable\ComfyUI\execution.py", line 83, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "C:\AI\ComfyUI_windows_portable\ComfyUI\execution.py", line 76, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "C:\AI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-MotionDiff\smpl_nodes.py", line 53, in convent
thetas, meta = convention.joint2smpl(joints)
File "C:\AI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-MotionDiff\mogen\smpl\simplify_loc2rot.py", line 101, in joint2smpl
new_opt_cam_t, new_opt_joint_loss = _smplify(
File "C:\AI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-MotionDiff\mogen\smpl\joints2smpl\src\smplify.py", line 186, in __call__
camera_optimizer.step(closure)
File "C:\AI\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\optim\optimizer.py", line 280, in wrapper
out = func(*args, **kwargs)
File "C:\AI\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
return func(*args, **kwargs)
File "C:\AI\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\optim\lbfgs.py", line 312, in step
orig_loss = closure()
File "C:\AI\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
return func(*args, **kwargs)
File "C:\AI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-MotionDiff\mogen\smpl\joints2smpl\src\smplify.py", line 182, in closure
loss.backward()
File "C:\AI\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\_tensor.py", line 487, in backward
torch.autograd.backward(
File "C:\AI\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\autograd\__init__.py", line 200, in backward
Variable._execution_engine.run_backward( # Calls into the C++ engine to run the backward pass

**Comment by NeedsMoar** [comment]
It's likely that AMD / torch_directml doesn't work with autograd given how many limitations the DirectML backend has.
